pages/common/inkmake.md

```diff
@@ -0,0 +1,24 @@
+# inkmake
+
+> Exports from SVG files using Inkscape's backend.
+> More information: <https://rubygems.org/gems/inkmake>.
+
+- Display help:
+
+`./inkmake -h`
```

Comment:
Is there a reason the binary is in a relative path, instead of being in the user's `$PATH`?

Reply:
> Is there a reason the binary is in a relative path, instead of being in the user's `$PATH`?

Its installable with ruby gem so...

Reply:
Hm, let's see what @bl-ue or @sbrl think.

Reply:
Let's just use `inkmake`.  tldr-pages examples assume the demonstrated command is on `$PATH`

Reply:
OK